src/ModelContextProtocol/Client/McpClient.cs

```diff
@@ -41,7 +41,7 @@ public McpClient(IClientTransport transport, McpClientOptions options, McpServer
             }
 
             SetRequestHandler<CreateMessageRequestParams, CreateMessageResult>(
-                "sampling/createMessage",
+                RequestMethods.SamplingCreateMessage,
```

Comment:
Do we need to add these consts? And if so, do we have to use them? This reminds me of using `ClaimConstants.Scp` instead of just `"scp"`. This isn't quite that extreme, but it is more verbose than before and doesn't help me when scanning the code and the spec at the same time.

Reply:
I think we want them. These aren't tiny strings like "scp". I've gotten them wrong on multiple occasions and had to debug why, e.g. "tool/list" instead of "tools/list". It's much easier when you get a compilation failure when there's a mistake, and when IntelliSense completes it for you.